js/extension.js

```diff
@@ -86,6 +86,8 @@ function computeId(task, config) {
         for (var arg of args) {
             if (typeof arg == "string") {
                 props.push(arg);
+            } else if (typeof arg == "object") {
```

Comment:
when arg is object,this function get a wrong taskID witch diffirent from the taskID below